Source Main 5.2/source/WSclient.cpp

```diff
@@ -218,8 +218,6 @@ int MoveCount = 0;
 
 int CurrentSkill = 0;
 
-bool Teleport = false;
```

Comment:
This is the root of all evil. Better to remove it. In the server side, this is tied to the player, while here, it is used for everything that is teleporting.